src/pyhf/cli/spec.py

```diff
@@ -122,3 +122,132 @@ def inspect(workspace, output_file, measurement):
         with open(output_file, 'w+') as out_file:
             json.dump(result, out_file, indent=4, sort_keys=True)
         log.debug("Written to {0:s}".format(output_file))
+
+
+@cli.command()
+@click.argument('workspace', default='-')
+@click.option(
+    '--output-file',
+    help='The location of the output json file. If not specified, prints to screen.',
+    default=None,
+)
+@click.option('-c', '--channel', default=[], multiple=True, metavar='<CHANNEL>...')
+@click.option('-s', '--sample', default=[], multiple=True, metavar='<SAMPLE>...')
+@click.option('-m', '--modifier', default=[], multiple=True, metavar='<MODIFIER>...')
+@click.option(
+    '-t',
+    '--modifier-type',
+    default=[],
+    multiple=True,
+    type=click.Choice(modifiers.uncombined.keys()),
+)
+@click.option(
+    '-e', '--measurement', default=[], multiple=True, metavar='<MEASUREMENT>...'
```

Comment:
why `e`? This doesn't seem super intuitive to me. I realize that `m` is already taken.